Azure Sentinel fix for outgoing mirroring conflicts (#29900)
demisto/content: Packs/AzureSentinel/ReleaseNotes/1_5_24.md

#### Integrations

##### Microsoft Sentinel

> - Fixed an issue where outgoing mirroring changes did not update the remote incidents if they were not synced in XSOAR.

Reviewer: ```suggestion
- Fixed an issue where outgoing mirroring changes did not update incidents in Azure Sentinal if those were not synced first into XSOAR.
```
